### ISSUE TITLE
Protocol ver 3.4

### ISSUE THREAD

**Issue by grosjo** [EmilSodergren/rust-tuyapi#29]
Hi

Is your livrary compatible with 3.4 ?

Thank you

**Comment by EmilSodergren** [comment]
Nope, not yet. It is on my todo-list, as well as 3.5, but I really do not have any time to work on this now.
